lib/galaxy/model/__init__.py

```diff
@@ -792,6 +792,23 @@ def __init__(self, email=None, password=None, username=None):
         self.active = False
         self.username = username
 
+    def get_user_data_tables(self, data_table: str):
+        session = object_session(self)
```

Comment:
```suggestion
        session = object_session(self)
        assert session
```
Otherwise mypy will complain on `session.execute()` since it can be None